NWEEI/NWEEI/Views/Resources/Index.cshtml

```diff
@@ -1,5 +1,133 @@
-﻿@*
-    For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
-*@
+﻿@model NWEEI.ViewModels.SearchViewModel
+@using System.Text.RegularExpressions;
+
 @{
+    ViewData["Title"] = "Resources";
 }
+
+<!-- search -->
+<div class="dark-block search-container">
+    <div class="container">
+        <div class="content-section" style="line-height:1.5em;">
+
+            <h4 class="text-center">Search NWEEI</h4>
+
+            <form asp-action="Index">
+                <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+                <div class="form-group search-bar">
+                    <!-- search input and icon -->
+                    <div class="search-bar-content">
+                        <!-- search input -->
+                        <input asp-for="SearchQuery" class="form-control search-bar" />
+
+                        <!-- icon -->
+                        <span class="search-submit">
+                            <button type="submit" style="background: none; color: inherit; border: none; padding: 0; font: inherit; cursor: pointer; outline: inherit; position: absolute;">
+                                <i type="submit" class="fa fa-search"></i>
+                            </button>
+                        </span>
+                    </div>
+                </div>
+            </form>
+
+
+        </div>
+    </div>
+</div>
+
+<!-- results -->
+<div class="container">
+    @if (Model.HasResults)
+    {
+        <div>
+            <h6>
+                Your search for "@Model.SearchQuery" returned
+                <a href="#article-results">@Model.Articles.Count articles</a>
+                and <a href="#faq-results">@Model.FAQs.Count FAQs</a>.
+            </h6>
+            <div>
+            </div>
+        </div>
+    }
+
```

Comment:
It'd be nice to have an `else` here that yielded a 'no results found' notification.